build-tools/_build-lib.sh

```diff
@@ -77,7 +77,7 @@ ginkgo_test_with_coverage () {
   (
     export GOBIN="$BUILDDIR/bin"
     echodo cd "$WKDIR/src/$PKGIMPORT"
-    echodo ginkgo -r -keepGoing -trace -randomizeAllSpecs -progress --nodes 4 -cover
+    echodo ginkgo -r -compilers 1 -keepGoing -trace -randomizeAllSpecs -progress --nodes 4 -cover
```

Comment:
remove the compilers flag, this isn't an issue for us currently and we need more investigation as to whether we need to limit this.